src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.tns.html

```diff
@@ -0,0 +1,8 @@
+<StackLayout class="segment-example">
+        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)"
```

Comment:
Use naming convention:
```suggestion
        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)"
```

Reply:
Done